_posts/2020-09-01-processing-java-stream.md

```diff
@@ -0,0 +1,128 @@
+---
+layout:    post
+title:     "Java Stream - przetwarzanie elementów"
+date:      2020-09-01 08:00:00 +0100
+published: true
+author:    tdrag
+image:     /assets/img/posts/2020-09-01-java-stream-processing/waterfall-stream.jpg
+tags:
+    - java
+    - stream
+    - benchmarking
+---
+
+# Java Stream
```

Comment:
W tekście nie stosujemy nagłówków pierwszego rzędu - to jest zarezerwowane na tytuł i dodawane z automatu przez jekylla. Zmień proszę nagłówki na drugiego rzędu